README.md

```diff
@@ -121,8 +121,8 @@ you should see something like this:
 
     Performing system checks...
     System check identified no issues (0 silenced).
-    September 02, 2016 - 01:46:02
-    Django version 1.10.1, using settings 'usaspending_api.settings'
+    April 17, 2017 - 01:46:02
+    Django version 1.11.0, using settings 'usaspending_api.settings'
```

Comment:
@willkjackson wow, that's pretty thorough to catch a change in our sample output...thanks!